### ISSUE TITLE
Unsafe suggestions

### ISSUE THREAD

**Issue by pdcmoreira** [p42ai/js-assistant#61]
I've just installed this extension and the first suggestion I got raised a huge red flag for me, as it **changes the output** of the function instead of just the syntax.

The suggestion is the following:
<img width="493" alt="image" src="https://user-images.githubusercontent.com/4430606/203043554-19d6a53b-9720-4844-8d9e-6c8c5dbe81aa.png">

But by applying that, changes the behavior of the function to null and '' (empty string) values!

```js
// Original
const funcA = a => {
  a = a || 'Default'
  
  return a
}

// Suggested change
const funcB = (a = 'Default') => {
  return a
}

;[undefined, null, '', 'test'].forEach(p => console.log(funcA(p) === funcB(p))) // outputs: true, false, false, true
```

Is there a way to disable unsafe suggestions?

**Comment by lgrammel** [comment]
@pdcmoreira thanks for your feedback!

I'll add a configuration option to turn off unsafe suggestions in the diagnostics and the side panel.

**Comment by lgrammel** [comment]
I've just released `v2.5.0` which contains a new configuration option to turn off unsafe suggestions:

![image](https://user-images.githubusercontent.com/205036/203751669-fe8a1b8b-9166-4035-ba78-72f0cc22f38b.png)

Thanks again for the idea!
